etc/bento.env

```diff
@@ -439,3 +439,19 @@ BENTO_CBIOPORTAL_SESSION_DATABASE_IMAGE_VERSION=6.0.7
 BENTO_CBIOPORTAL_SESSION_DATABASE_CONTAINER_NAME=${BENTOV2_PREFIX}-cbioportal-session-db
 BENTO_CBIOPORTAL_SESSION_DATABASE_DATA_DIR=${BENTO_FAST_DATA_DIR}/cbioportal/session_db
 # Uses BENTO_CBIOPORTAL_SESSION_NETWORK as the network
+
+# Monitoring
+BENTO_MONITORING_NETWORK=${BENTOV2_PREFIX}-monitoring-net
+BENTOV2_PRIVATE_GRAFANA_URL=${BENTOV2_PORTAL_PUBLIC_URL}/api/grafana
+BENTOV2_PUBLIC_GRAFANA_URL=${BENTOV2_PORTAL_PUBLIC_URL}/grafana
+BENTOV2_LOKI_BASE_IMAGE=grafana/loki
+BENTOV2_LOKI_BASE_IMAGE_VERSION=3.0.0
+BENTOV2_LOKI_CONTAINER_NAME=${BENTOV2_PREFIX}-loki
+BENTOV2_LOKI_PROD_TEMP_DIR=${BENTO_SLOW_DATA_DIR}/loki/tmp
+BENTOV2_GRAFANA_BASE_IMAGE=grafana/grafana
+BENTOV2_GRAFANA_BASE_IMAGE_VERSION=11.1.1-ubuntu
+BENTOV2_GRAFANA_CONTAINER_NAME=${BENTOV2_PREFIX}-grafana
+BENTOV2_GRAFANA_PROD_LIB_DIR=${BENTO_SLOW_DATA_DIR}/grafana/lib
+BENTOV2_PROMTAIL_BASE_IMAGE=grafana/promtail
+BENTOV2_PROMTAIL_BASE_IMAGE_VERSION=2.9.2
+BENTOV2_PROMTAIL_CONTAINER_NAME=${BENTOV2_PREFIX}-promtail
```

Comment:
make sure to add trailing newlines to your text files – it's required to be a valid text file in POSIX

Reply:
also, new environment variables should not be named with the `BENTOV2` prefix - use `BENTO` as a prefix instead